src/js/components/Menu/__tests__/Menu-test.js

```diff
@@ -302,6 +354,32 @@ describe('Menu', () => {
     expect(document.getElementById('test-menu__drop')).toBeNull();
   });
 
+  test('open on up close on esc', () => {
+    const { getByLabelText, container } = render(
+      <Grommet>
+        <Menu
+          id="test-menu"
+          label="Test"
+          items={[{ label: 'Item 1' }, { label: 'Item 2' }]}
+        />
+      </Grommet>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+
+    fireEvent.keyDown(getByLabelText('Open Menu'), {
+      key: 'Up',
+      keyCode: 38,
+      which: 38,
+    });
+    fireEvent.keyDown(getByLabelText('Close Menu'), {
```

Comment:
Can we put an assertion after the 'Up' keyDown to check that the Menu is being opened

Reply:
Good idea, about to push the changes to my fork